src/actions/templates.js

```diff
@@ -225,8 +226,13 @@ export const open = () => dispatch => {
     const filename = filenames[0]
     if (filename.split('.').pop() !== 'mjml') { return }
 
+    const isWin = (/^win/.test(process.platform))
+    const directorySplit = isWin ? '\\' : '/'
+    const templateName = filename.split(directorySplit).pop()
+        .split('.').shift()
+
```

Comment:
It would be simpler to use the [path.basename()](https://nodejs.org/api/path.html#path_path_basename_path_ext) method:

```
const filename = path.basename(filenames[0], '.mjml')
```